### DISCUSSION TITLE
RefactoringUI site

### DISCUSSION THREAD

**Discussion by ItsTerm1n4l** [tailwindlabs/tailwindcss#14033] [Help] [answered]
Hello. this is probably the wrong place to ask this but I coudn't find anywhere better.
how did you get these lines that connect divs?
![20240722_07h53m53s_grim](https://github.com/user-attachments/assets/cc5f2a46-3fc9-4679-a3f1-427963fbb2ca)
from https://www.refactoringui.com/
Thank you!

**Comment by wongjn** [comment, marked as answer]
You can use the *Inspect Element* tool in your browser's developer tools to take a look at how they were built:

```html
<div class="absolute top-[70%] right-[68%] h-[24vw] min-h-[8rem] w-[28vw] rounded-tl-xl border-l-2 border-t-2 border-sky-400 sm:top-[79.37336815%] sm:right-[78.32898172%] sm:h-24 sm:min-h-0 sm:w-auto sm:rounded-tl-none sm:border-x sm:border-t-0 lg:top-[39.16449086%] lg:right-[60.83550914%] lg:h-6 lg:w-[21.4375rem] lg:rounded-tl-xl lg:border-t-2 lg:border-r-0 lg:border-l-2">
  <svg …>…</svg>
  <div class="absolute -left-6 top-full mt-1 whitespace-nowrap font-mono font-medium leading-8 text-slate-700 sm:-left-9 lg:-left-4">.primary {<br>fill: #637985; }</div></div>
```
The lines are drawn using [CSS borders](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Backgrounds_and_borders#borders), and positioned using [absolute positioning](https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Positioning#absolute_positioning).

**Reply by ItsTerm1n4l** [reply]
Is there no JS involved then?

**Reply by wongjn** [reply]
It does not seem like there is any JS involved.

**Reply by ItsTerm1n4l** [reply]
Thank you for your help
